lib/evaluation/evaluationClient.ts

```diff
@@ -69,12 +64,20 @@ export class EvaluationClient {
     this._did = did;
   }
 
-  public get verifiablePresentation(): VerifiablePresentation {
-    return this._verifiablePresentation;
+  public get presentationSubmission(): PresentationSubmission {
+    return this._presentationSubmission as PresentationSubmission;
   }
 
-  public set verifiablePresentation(verifiablePresentation: VerifiablePresentation) {
-    this._verifiablePresentation = verifiablePresentation;
+  public set presentationSubmission(presentationSubmission: Partial<PresentationSubmission>) {
+    this._presentationSubmission = presentationSubmission;
+  }
+
+  public get verifiableCredential(): VerifiableCredential[] {
+    return this._verifiableCredential as VerifiableCredential[];
+  }
+
+  public set verifiableCredential(verifiableCredential: Partial<VerifiableCredential>[]) {
```

Comment:
it's not outside world. we're exposing our methods in pejs.ts file

Reply:
The Partial as a property setter shouldn't be there IMO, that is the point. Afaik we aren't passing in Partials. It it there because the VC might not be set/populated in the beginning right @maikel-maas 

So why not convert the argument to a partial to begin with?

Reply:
It's a partial already. I have to keep it like that because the strict mode enforces that VerifiableCredential cannot be assigned to VerifiableCredential | undefined (Partial).

Reply:
What doesn't work about the below solution?

```
vcs: Partial<VerifiableCredential[]>

public setVerifiableCredential (vcs: VerifiableCredential[]) {
  this.vcs = vcs
}

```

Reply:
Misunderstanding, fixed.